kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -185,25 +185,23 @@
               </KButton>
               <KCircularLoader v-if="t.id === subTopicLoading" />
             </div>
-            <!-- search results -->
+            <!-- resources that are not in a folder -->
             <HybridLearningCardGrid
               v-if="resources.length"
-              :contents="resources"
+              :contents="resourcesDisplayed"
               :numCols="numCols"
               :genContentLink="genContentLink"
               cardViewStyle="card"
               @toggleInfoPanel="toggleInfoPanel"
             />
-            <div v-if="topicMore" class="end-button-block">
```

Comment:
Deleting this completely means that if there are sufficient resources in the top level of the topic to necessitate server side pagination, we will now never be able to load them.

We need both this button and the other button.